org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/internal/Logger.java

```diff
@@ -23,16 +25,22 @@
      */
     public static void logInfo(String message) {
         if (isVerbose) {
-            JavaDebuggerServerPlugin.logInfo(message);            
+            JavaDebuggerServerPlugin.logInfo(message);
+            inner.info(message);
         }
     }
 
     public static void logException(String message, Exception e) {
         JavaDebuggerServerPlugin.logException(message, e);
+        inner.error(message, e);
     }
 
     public static void logError(String error) {
         JavaDebuggerServerPlugin.logError(error);
+        inner.error(error);
     }
     
+    public static void logWarn(String warn) {
+        inner.warn(warn);

```

Comment:
I think JavaDebuggerServerPlugin should log some thing(eg: logInfo if there is no warn method).